core/schema/schema.go

```diff
@@ -28,7 +30,7 @@ type SchemaFile struct {
 }
 
 type Repository interface {
-	Create(ctx context.Context, namespace string, schema string, metadata *Metadata, versionID string, schemaFile *SchemaFile) (version int32, err error)
+	Create(ctx context.Context, namespace string, schema string, metadata *Metadata, versionID string, schemaFile *SchemaFile, commitSHA string) (version int32, err error)
```

Comment:
Too many parameters to the function (7), can we check if we can reduce their number.

Reply:
addressed